scripts/install.sh

```diff
@@ -104,9 +104,9 @@ set -o errexit
 set -o nounset
 
 # sphinxdoc-install-invenio-full-begin
-# FIXME needed for invenio-search
+# FIXME the next command needed until https://github.com/inveniosoftware/invenio-search/pull/34 is released on PyPI
```

Comment:
Done.  I have also updated to the latest `invenio-records-ui`, `invenio-theme` and friends.